posthog/api/survey.py

```diff
@@ -462,6 +462,15 @@ class Meta:
         ]
         read_only_fields = fields
 
+    # We had a user write in complaining that we were exposing the description in the API
+    # (https://posthoghelp.zendesk.com/agent/tickets/15210), which was a problem for them
+    # since they were using it as a way to store sensitive information. Given that we don't ever use
+    # that field to render the survey, we can safely remove it from the API response.
+    def to_representation(self, instance):
```

Comment:
I'm probably missing something here, but can we get the same result by just removing the `description` property from the serialized properties list here? 

https://github.com/PostHog/posthog/blob/6d48bcff16e2854582e7825bd72bed045bb4ec49/posthog/api/survey.py#L450

Reply:
you right.  I'm doing that instead.